manifests/cf-manifest/spec/support/manifest_helpers.rb

```diff
@@ -68,20 +100,30 @@ def workdir
     Cache.instance.workdir ||= $workdir
   end
 
+  def manifest_without_vars_store
```

Comment:
This doesn't seem to be used anywhere

Reply:
I added a test in `manifests/cf-manifest/spec/manifest/rotate_certs_spec.rb`, I forgot to push, sorry.